docker-compose.yml

```diff
@@ -30,9 +30,18 @@ services:
       - "6080:6080"
 
   gargoyle-sts:
-    image: kr7ysztof/gargoyle-sts:0.0.2
+    image: kr7ysztof/gargoyle-sts:0.0.3
     environment:
       - STS_HOST=0.0.0.0
       - STS_PORT=12345
+      - GARGOYLE_KEYCLOAK_PUBLIC_KEY_ID=FJ86GcF3jTbNLOco4NvZkUCIUmfYCqoqtOQeMfbhNlE
```

Comment:
Gargoyle-sts image currently expects the misspelled env vars (KEYKLOAK instead of KEYCLOAK). We should adjust the gargoyle-sts image